script/Ciel/Day 6/06_01_CIEL06_6.txt

```diff
@@ -148,151 +157,162 @@ Lying there in bed,
 }
 [sha:f12fa985a92c93aa1b95c162c09541550f2ea47f]{
 -- Page 242, Offset 13786.
 -- 単一種としての永久機関なら、遙かな昔から完璧に近い生物が存在する。
-Through achieving perpetual motion, there exists, since the distant past, a species far closer to perfection.
+// Hella tight page
+If there is a species that has approached perpetuity,
+ it is one that has existed since time immemorial.
 }
 [sha:0f43c39043f7e3514b27c49221568dbdde1f6348]{
 -- Page 242, Offset 13787.
 -- 自らの体を食料とし、それを糧に繁殖する。
-They feed and reproduce by consuming their own bodies.
+One that uses its own body as a source of food,
+ as well as a source of energy for reproduction.
 }
 [sha:9a5003930efafa280b095e36268aff9af8bc700a]{
 -- Page 242, Offset 13788.
 -- 寿命などというものもない。古くなった細胞は栄養源として食料になり、新しい細胞を生み出し続ける。
-Notions like 'lifespan' do not concern them. Aging cells are a source of nutrients to feed upon, leading to the endless creation of new cells.
+The concept of a 'lifespan' doesn't even apply to them.
+ They recycle their own aging cells as a source of nutrition,
+ and use that food to produce new cells.
 }
 [sha:33241f1e16bc4f1a54e8295394e74e647a0a6671]{
 -- Page 242, Offset 13789.
 -- 群体と呼ばれるもの。例えば<海月|クラゲ>。
-They're referred to as colonial organisms. Jellyfish are an example of such organisms.
+Colonial organisms. Jellyfish are one such example.
 }
 [sha:1f257b0e7a4dd56fc905c8a271a8bf0331aebabc]{
 -- Page 242, Offset 13790.
 -- しかし、それは知性という余分な機能を持たないが故の永久機関だ。知性を持たないでいいと言うのなら、それは死をもって永遠とする結論となんら変わりはない。
 // this is bullshit
-However, they're perpetual organisms exactly because they lack the extraneous faculty that is reason. Consenting to not having reason is no different than coming to the conclusion that eternity is death.
+// This is putting ME to sleep
+However, they're perpetual organisms precisely because they lack the extraneous faculty known as reason.
+ To forsake reason is no different than to conclude that death itself represents eternity.
 }
 [sha:7fcd4fc72437c37da34b3dfa02cdbf8accc1ad8f]{
 -- Page 243, Offset 13791.
 -- ヒトとしての形を保ちながら永遠でいたいのならば、不老不死という手段では不可能だ。
-It is impossible to reach eternity while maintaining human form through the method known as 'immortality'.
+It is impossible to exist forever while simultaneously maintaining your form as a human,
+ at least via the method known as 'immortality'.
 }
 [sha:9e63debfa60e10f817407d082ec27aa5dffaa096]{
 -- Page 243, Offset 13792.
 -- 長い年月は肉体を崩壊させ、思考精度を萎縮させる。
-The passage of time causes the collapse of bodies and the withering of mental faculties.
+The relentless march of time causes bodies to crumble, and mental faculties to diminish.
 }
 [sha:6f9e15152e1b003a08caa82da9c51c29000a732d]{
 -- Page 243, Offset 13793.
 -- 不老不死か、永遠か。
-Immortality? Eternity?
+Immortality? Or eternity?
 }
 [sha:5bdec41f037be4f43fdefab3e69b366c80a8b92d]{
 -- Page 243, Offset 13794.
 -- 手垢のついた不老不死になど未練はない。
-I have no lingering attachment to this hackneyed notion of immortality.
+I have no lingering attachment to the hackneyed notion of immortality.
```

Comment:
prefer this

Reply:
My rationale here is based on how I interpreted the context - it reads to me as though he's weighing 'immortality' vs 'eternity' as disparate things, and declares that eternity is the better of the two. Using `this` read slightly more to me like he's lumping _both_ things he's talking about together. He's kinda holding the old busted idea of immortality at arms length, so I might also lean towards `the` because it feels less 'proximate' as a word than `this`.